EnglishFilter/sections/specific.txt

```diff
@@ -327,6 +327,8 @@ goal.com##div[class^="layout_banner__"]
 goal.com##aside[class^="layouts_asideContentCommercial__"]
 dramacool.*##.sidebar.mCS-autoHide
 who.is##.home-video-blurb
+healthline.com##.article-body > div[class] > div > p.disclaimer
+healthline.com##a[href^="https://activation.healthline.com/api/member-offers/"]
```

Comment:
Unsure about this rule on main page (banner at top).

Reply:
I don't see any other a with such href on the website too, removed.